storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -953,6 +954,20 @@ void Archived_Redo_Log_Monitor::thread_func() {
     std::string dir;
     std::string subdir;
   } archive;
+  std::ostringstream query;
+  read_mysql_variables(mysql, "SHOW VARIABLES", vars, true);
+
+  if (!(redo_log_archive_dirs == nullptr || *redo_log_archive_dirs == 0)) {
+    xb::info() << "innodb_redo_log_archive_dirs is now  set .";
+    redo_arch = true;
+  }
+
+  if (xtrabackup_redo_log_arch_dir) {
```

Comment:
Here we are setting redo archive unconditionally, what happens if someone already have it set? we will overwrite to --redo-log-arch-dir and later return it back to NULL. We should only set if it is not already set.